ucte-network/src/test/resources/20170322_1844_SN3_FR2.uct

```diff
@@ -7,11 +7,11 @@ License, v. 2.0. If a copy of the MPL was not distributed with this
 file, You can obtain one at http://mozilla.org/MPL/2.0/.
 ##N
 ##ZFR
-FFNGEN71 FNGEN__ GEN- 0 2 24.000 0.00000 0.00000 -800.00 0.00000 9999.00 -9999.0 999.000 -999.00 
-FFNHV111 FNHV1__ HV1- 0 0        0.00000 0.00000 0.00000 0.00000 
```

Comment:
is it a good idea to trick the input test file (strip the space chars at the end) ?